root/layout/components/BottomMenu.js

```diff
@@ -251,6 +251,9 @@ const EditingMenu = () => (
       <li>
         <a href="/reports">{l('Reports')}</a>
       </li>
+      <li>
+        <a href="/voting-reports">{l('Voting Reports')}</a>
```

Comment:
Also, _Voting Reports_ might be misleading as these are searches rather than daily built reports like (data) _Reports_.